build.py

```diff
@@ -345,6 +348,7 @@ def file_lookup(name):
 
       code = HEADER + "\n" + json_data["compiledCode"]
       code = code.replace(remove, "")
+      code = code.replace('Blockly.version="unset"', 'Blockly.version="' + blocklyVersion + '"')
```

Comment:
This is rather fragile.  I would do the replacement on the code *before* compilation, since you've got now guarantees of what you'll get out the other side.  Any of these (and more) are possible:
```
Blockly.version="unset"
Blockly.version='unset'
Blockly.c="unset"
f.c="unset"
var fc="unset"
var Blockly={version:"unset",...}
```

Reply:
Yeah that makes sense.